### ISSUE TITLE
React component not working!

### ISSUE THREAD

**Issue by NityaSG** [FlowiseAI/FlowiseChatEmbed#21]
while installing the library for React component : 
import { FullPageChat } from "flowise-embed-react"

npm i flowise-embed-react

i am getting the error : 
Can't resolve 'flowise-embed/dist/web'

**Comment by HenryHengZJ** [comment]
Yeah so this error can be seen if you are using [CRA](https://github.com/facebook/create-react-app/issues/11865)

This can be overcome using this [fix](https://stackoverflow.com/questions/70964723/webpack-5-in-ceate-react-app-cant-resolve-not-fully-specified-routes/75109686#75109686)
